presto-main/src/main/java/io/prestosql/execution/scheduler/UniformNodeSelector.java

```diff
@@ -262,7 +262,7 @@ private void equateDistribution(Multimap<InternalNode, Split> assignment, NodeAs
             InternalNode maxNode = maxNodes.poll();
             InternalNode minNode = minNodes.poll();
 
-            if (assignmentStats.getTotalSplitCount(maxNode) - assignmentStats.getTotalSplitCount(minNode) <= 1) {
+            if (assignmentStats.getTotalSplitCount(maxNode) - assignmentStats.getTotalSplitCount(minNode) <= 5) {
```

Comment:
Add a comment on what this does and why `5` is important.

Reply:
+1

Reply:
I've added explanation in commit message plus comment in code.